topics/dev/tutorials/toolshed/slides.html

```diff
@@ -83,9 +83,9 @@
 
 - Only for third party toolsheds (Main and Test by default)
 - In `config/tool_sheds_conf.xml`:
-```bash
+```xml
   <tool_sheds>
-    <tool_shed name="Galaxy main tool shed" url="https://toolshed.g2.bx.psu.edu/" />
+    <tool_shed name="Galaxy main tool shed" url="https://toolshed.g2.bx.psu.edu/#" />
   </tool_sheds>
 ```
```

Comment:
We can probably just drop this example, since it's just (part of) the default config file.

Reply:
That works too. If you're fine with that I'll kill that

Reply:
Go ahead!